docs/src/main/sphinx/connector/mariadb.rst

```diff
@@ -71,41 +71,167 @@ that catalog name instead of ``mariadb`` in the above examples.
+to the following table:
+
+.. list-table:: MariaDB type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - MariaDB type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``TINYINT``
+    - ``BOOL`` and ``BOOLEAN`` are aliases of ``TINYINT(1)``
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``INT``
+    - ``INTEGER``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``DECIMAL(p,s)``
+    - ``DECIMAL(p,s)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``TINYTEXT``
+    - ``VARCHAR(255)``
+    -
+  * - ``TEXT``
+    - ``VARCHAR(65535)``
+    -
+  * - ``MEDIUMTEXT``
+    - ``VARCHAR(16777215)``
+    -
+  * - ``LONGTEXT``
+    - ``VARCHAR``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``TINYBLOB``
+    - ``VARBINARY``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    -
+  * - ``MEDIUMBLOB``
+    - ``VARBINARY``
+    -
+  * - ``LONGBLOB``
+    - ``VARBINARY``
+    -
+  * - ``VARBINARY(n)``
+    - ``VARBINARY``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIME(n)``
+    - ``TIME(n)``
+    -
+  * - ``TIMESTAMP(n)``
+    - ``TIMESTAMP(n)``
+    - MariaDB stores the current timestamp by default. Please enable
+      `explicit_defaults_for_timestamp <https://mariadb.com/docs/reference/mdb/system-variables/explicit_defaults_for_timestamp/>`_
+      to avoid implicit default values.
+
+No other types are supported.
+
+Trino type mapping to MariaDB type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Trino types to the corresponding MariaDB types according
+to the following table:
+
+.. list-table:: Trino type mapping to MariaDB type mapping
+  :widths: 30, 25, 50
+  :header-rows: 1
+
+  * - Trino type
+    - MariaDB type
+    - Notes
+  * - ``TINYINT``
+    - ``BOOLEAN``
+    -
+  * - ``TINYINT``
+    - ``TINYINT``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``INTEGER``
+    - ``INT``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``REAL``
+    - ``FLOAT``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``DECIMAL(p,s)``
+    - ``DECIMAL(p,s)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``VARCHAR(255)``
+    - ``TINYTEXT``
+    - Maps on ``VARCHAR`` of length 255 or less.
+  * - ``VARCHAR(65535)``
+    - ``TEXT``
+    - Maps on ``VARCHAR`` of length between 256 and 65535, inclusive.
+  * - ``VARCHAR(16777215)``
+    - ``MEDIUMTEXT``
+    - Maps on ``VARCHAR`` of length between 65536 and 16777215, inclusive.
+  * - ``VARCHAR``
+    - ``LONGTEXT``
+    -
+  * - ``MEDIUMBLOB``
+    - ``MEDIUMBLOB``
+    -
+  * - ``VARBINARY``
+    - ``TINYBLOB``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIME(n)``
+    - ``TIME(n)``
+    -
+  * - ``TIMESTAMP(n)``
+    - ``TIMESTAMP(n)``
+    - MariaDB stores the current timestamp by default. Please enable
```

Comment:
Avoid "Please" and other filler words

Reply:
@mosabua I am aware of that style guide standard but wasn't sure how much I was supposed to change given that the PR should not fix unrelated issues (minor copy edits). I'll make the change since you suggested it.